Sources/Build/BuildPlan.swift

```diff
@@ -542,16 +553,31 @@ public final class SwiftTargetBuildDescription {
     /// These are the source files derived from plugins.
     private var pluginDerivedSources: Sources
 
+    /// These are the resource files derived from plugins.
+    private var pluginDerivedResources: [Resource]
+
     /// Path to the bundle generated for this module (if any).
     var bundlePath: AbsolutePath? {
-        buildParameters.bundlePath(for: target)
+        if target.underlyingTarget.bundleName != nil {
+            return buildParameters.bundlePath(for: target)
+        } else if !pluginDerivedResources.isEmpty {
+            // FIXME: Use proper bundle name here.
+            return buildParameters.bundlePath(named: "foo_\(target.name)")
```

Comment:
How come asking for the target's bundle doesn't work here?

Reply:
Currently, the bundle name is set to `nil` when there are no resources during the initial package construction, I'll be changing that.

Reply:
Ah, cool.  That sounds like a good approach.